### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by AlexeyAbdulmanov**

### CODE REVIEW

File: app/views/layouts/application.html.slim [hunk 1]
```diff
@@ -1,6 +1,14 @@
 doctype html
 html[lang="en"]
   head
+    script[type="text/javascript"]
```
Comment (fs-admin) [inline app/views/layouts/application.html.slim]: pls put it into file.

File: app/views/layouts/application.html.slim [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+      |  (function(m,e,t,r,i,k,a){m[i]=m[i]||function(){(m[i].a=m[i].a||[]).push(arguments)}; m[i].l=1*new Date();k=e.createElement(t),a=e.getElementsByTagName(t)[0],k.async=1,k.src=r,a.parentNode.insertBefore(k,a)}) (window, document, "script", "https://mc.yandex.ru/metrika/tag.js", "ym"); ym(52661314, "init", { clickmap:true, trackLinks:true, accurateTrackBounce:true });
+    noscript
+      div
+        img[src="https://mc.yandex.ru/watch/52661314" style="position:absolute; left:-9999px;" alt=""]
+    script[async src="https://www.googletagmanager.com/gtag/js?id=UA-135652191-1"]
```
Comment (fs-admin) [inline app/views/layouts/application.html.slim]: do not commit keys) UA-135652191-1

Comment (fs-admin) [reply app/views/layouts/application.html.slim]: script[async src="https://www.googletagmanager.com/gtag/js?id=#{ENV["GOOGLE_KEY"]}"]

File: app/views/posts/_form.html.slim [hunk 3]
```diff
@@ -6,8 +6,9 @@
     = f.text_field :title, class: 'form-control'
   .form-group
     = f.label :Текст
-    = f.text_field :text, class: 'form-control'
+    = f.cktext_area :text, class: 'form-control'
+  .form-check
+    = f.check_box :published, class: 'form-check-input'
+    = f.label :Опубликовать
```
Comment (fs-admin) [inline app/views/posts/_form.html.slim:12]: : лишнее

File: app/views/posts/_post.html.slim [hunk 4]
```diff
@@ -1,17 +1,23 @@
...
-            small
-            = post.user.email
-          h3
-            small
-              = l post.created_at, format: :short
-          h4
-            comment
-              = post.decorate.comments_count.to_s + " comments"
+- posts.each do |post|
+  -if post.published == true
```
Comment (fs-admin) [inline app/views/posts/_post.html.slim:2]: - if post.published

File: app/views/posts/_post.html.slim [hunk 5]
```diff
@@ -1,17 +1,23 @@
...
+                = link_to post.title, post_path(post)
+              h3
+                small
+                = post.user.email
+              h3
+                small
+                  = l post.published_at, format: :short, default: '-'
+              h4
+                comment
+                  -if post.comments_count != 0
```
Comment (fs-admin) [inline app/views/posts/_post.html.slim:19]: -if post.comments_count.positive?

File: app/views/posts/show.html.slim [hunk 6]
```diff
@@ -12,20 +12,18 @@
         h3
           = l post.created_at, format: :short
         p.text-justify
-          = post.text
+          = raw post.text
```
Comment (fs-admin) [inline app/views/posts/show.html.slim:15]: Нужно убедится, что там НЕ будет JS кода

File: app/views/users/_post.html.slim [hunk 7]
```diff
@@ -0,0 +1,17 @@
+h3
+  | Опубликованые
+- user.posts.each do |post|
```
Comment (fs-admin) [inline app/views/users/_post.html.slim:3]: model Post
  scope :published, -> { where(published: true) }
end

- user.posts.published.each do |post|

**Comment by AlexeyAbdulmanov** [comment]
added jquery-rails
Added Google and Yandex analytics
Create Readme
Ckeditor for Post ready
Add published and published_at to Post
i18n localization to ru
Added first and last name to user
Create base_decorator
Created counter_cache for comments
gem simplecov ready
Created custom url for Posts by friendly_id

**Pull request closed without merging**